python/ray/_private/utils.py

```diff
@@ -2001,3 +2001,18 @@ def validate_actor_state_name(actor_state_name):
             'it must be one of the following: "DEPENDENCIES_UNREADY", '
             '"PENDING_CREATION", "ALIVE", "RESTARTING", or "DEAD"'
         )
+
+
+def get_current_node_cpu_model_name() -> Optional[str]:
+    if not sys.platform.startswith("linux"):
```

Comment:
Other OSs need different way to get cpu model and they are non-trivial. Since our Ray clusters are mostly Linux so I didn't spend time to support other OSs (ROI is low).